packages/compiler-core/src/transforms/transformSlotOutlet.ts

```diff
@@ -68,9 +69,19 @@ export function processSlotOutlet(
   const propsWithoutName = name
     ? node.props.filter(p => p !== name)
     : node.props
+
   if (propsWithoutName.length > 0) {
+    //#2488
+    propsWithoutName.forEach(prop => {
```

Comment:
I think we need to check the type of the prop is AttributeNode or DirectiveNode, if the prop is DirectiveNode, we chould camelize content.

```
propsWithoutName.forEach(prop => {
  if (prop.type === NodeTypes.DIRECTIVE) {
    props.arg.content = camelize(arg.content)
  }
})
```